english/duck/duck.md

```diff
@@ -9,4 +9,7 @@ Wait, what was that noise?
 
 Large bats fly over your head.
 
-[Investigate](../cave/cave.md)
\ No newline at end of file
+A bit of LSD floats between your legs.
+
+[Investigate](../cave/cave.md)
+[Smash dat LSD](https://github.com/mattwoolz/create-your-own-adventure/blob/master/english/batmetal/batmetal.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.